src/ORM/Queries/SQLConditionalExpression.php

```diff
@@ -283,6 +283,10 @@ public function getJoins(&$parameters = array())
             // array('type' => 'inner', 'table' => 'SiteTree', 'filter' => array("SiteTree.ID = 1",
             // "Status = 'approved'", 'order' => 20))
             if (!is_array($join)) {
+                if (!empty($alias) && !is_numeric($alias) && $alias !== trim($join, '"')) {
```

Comment:
Should you have trimmed $alias in this comparison too?

Reply:
This is my fault. Framework is littered with "this table is quoted" and "this table must not be quoted" everywhere.

Reply:
> Should you have trimmed $alias in this comparison too?

Hmmn. I trimmed it after but it makes sense to move that above. I'll update it quickly